### ISSUE TITLE
[Request] Please provide PPA for (U)buntu distros

### ISSUE THREAD

**Issue by owais** [ishovkun/SierraBreeze#19]
It would be great to have this shipped through a Launchpad PPA for Ubuntu and Ubuntu based distros.

Thank you

**Comment by owais** [comment]
Can we at least leave this open in case someone from community wants to contribute?

**Comment by ishovkun** [comment]
Fair enough.

**Comment by thopiekar** [comment]
Yes, I can work on it. Got known of this project via G+.
A clear list of dependencies would be useful for the beginning.

**Comment by IgnusG** [comment]
```
depends=('kwin')
makedepends=('git' 'cmake' 'extra-cmake-modules')
```
That should do it @thopiekar. 

Taken from AURs package build

**Comment by thopiekar** [comment]
Done: https://code.launchpad.net/~thopiekar/+archive/ubuntu/sierrabreeze

**Comment by krisives** [comment]
@thopiekar Hey thanks for making the PPA. I wanted to add this to the README.md so that less savvy users could use it on Debian-based systems, but I wanted to check is this PPA going to be updated with changes from this git repository?

Thanks!

**Comment by thopiekar** [comment]
Yes, the packages get automatically build from master. At least within a day.

**Comment by krisives** [comment]
It seems they haven't been updated in 10 weeks or so? Maybe something changed or am I reading Launchpad wrong?

**Comment by thopiekar** [comment]
Good catch. In the build recipe I still had "build on request" set. Now it should build automatically, like I described before 😉👍

**Comment by krisives** [comment]
Thank you @thopiekar for your work!

**Comment by domsch1988** [comment]
Any Chance to get the ppa updated for 19.04?

**Comment by krisives** [comment]
For sure I will check it out and update this issue with results

**Comment by krisives** [comment]
I misread this it's @thopiekar that needs to update his PPA. I have a PPA for SierraBreezeEnhanced but he maintains the PPA for SierraBreeze itself and needs to click a few buttons in Launchpad to enable builds for Disco.
